applications/FluidDynamicsApplication/custom_processes/two_fluids_inlet_process.cpp

```diff
@@ -0,0 +1,243 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Simon Wenczowski
+//
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "utilities/openmp_utils.h"
+#include "utilities/variable_utils.h"
+
+// Application includes
+#include "two_fluids_inlet_process.h"
+#include "fluid_dynamics_application_variables.h"
+
+
+namespace Kratos
+{
+
+/* Public functions *******************************************************/
+
+TwoFluidsInletProcess::TwoFluidsInletProcess(
+    ModelPart& rModelPart,
+    Parameters& rParameters,
+    Process::Pointer p_dist_proc )
```

Comment:
I'd rename it to pDistanceProcess (to make it clearer and to follow the naming convention).

Reply:
Done